cmd/download_test.go

```diff
@@ -238,7 +238,7 @@ const payloadTemplate = `
 		"file_download_base_url": "%s",
 		"files": [
 			"/file-1.txt",
-			"/subdir/file-2.txt",
+			"/subdir/file-2#.txt",
```

Comment:
Directionally, this is great! Would you add a comment in the test setup to explain why the hash is part of the name? That's something that would easily be confusing a few weeks or months down the line, and it would be a bit tricky to figure out why it's there.